ScheduleParser/ParserResponse.cs

```diff
@@ -0,0 +1,10 @@
+namespace ScheduleParser;
+
+/// <summary>
+/// Class for deserializing <see cref="HttpContent"/>
+/// </summary>
+public class ParserResponse
+{
+    public string href { get; set; }
```

Comment:
```suggestion
    public string? href { get; set; }
```
А то nullability ругается. И чтобы имена свойств стайлгайд не нарушали (кстати, прикрутите StyleCop?), стоит их переименовать через JsonProperty или что-то такое